### PR TITLE
[New] Add PickMeUp@3.0.11 w/ git auto-update

### LINKED ISSUE #9314: [Request] Add pickmeup
**Library name:** pickmeup
**Git repository url:** https://github.com/nazar-pc/PickMeUp.git
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:** https://cleverstyle.org
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #9314
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/nazar-pc/PickMeUp
 * Official website (optional, not the repository): https://cleverstyle.org/
 * NPM package url (optional): https://www.npmjs.com/package/pickmeup
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 34
   - Count of stars: 527
   - Count of forks:  165
 * NPM download stats (optional):
   - Downloads in the last day: 13
   - Downloads in the last week: 293
   - Downloads in the last month: 791

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/pickmeup/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "url": "git://github.com/nazar-pc/PickMeUp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nazar-pc/PickMeUp.git",
+    "fileMap": [
+      {
+        "basePath": "css",
+        "files": [
+          "pickmeup*.+(css)"
```
Comment (sufuf3) [inline ajax/libs/pickmeup/package.json]: We don't use `css`as the bathPath.
Thanks

File: ajax/libs/pickmeup/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        ]
+      },
+      {
+        "basePath": "js",
```
Comment (sufuf3) [inline ajax/libs/pickmeup/package.json]: We don't use `js`as the bathPath.
Thanks

Comment (sashberd) [reply ajax/libs/pickmeup/package.json]: I think that those files are hosted by js pattern, do not them?
jquery.pickmeup.twitter-bootstrap.js and jquery.pickmeup.uikit.js

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

File: ajax/libs/pickmeup/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "name": "pickmeup",
+  "filename": "pickmeup.min.js",
+  "description": "Really simple, powerful, customizable and lightweight standalone datepicker",
+  "keywords": [
+    "datepicker",
+    "date",
+    "calendar"
+  ],
+  "version": "3.0.11",
+  "homepage": "https://cleverstyle.org/",
```
Comment (extend1994) [inline ajax/libs/pickmeup/package.json]: https://nazar-pc.github.io/PickMeUp/

File: ajax/libs/pickmeup/package.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      },
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "license": "MIT"
```
Comment (extend1994) [inline ajax/libs/pickmeup/package.json]: Updated to `0BSD` 29 days ago.
See https://github.com/nazar-pc/PickMeUp/blob/master/package.json#L17

File: ajax/libs/pickmeup/package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "pickmeup",
```
Comment (extend1994) [inline ajax/libs/pickmeup/package.json]: Let's use PickMeUp which is the name of GitHub repo for git auto-update here.
Don't forget to change folder name and run `npm test` before pushing the branch.

File: ajax/libs/pickmeup/package.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    "url": "git://github.com/nazar-pc/PickMeUp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nazar-pc/PickMeUp.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "{js,css}/?(jquery.)pickmeup*.+(js|css|map)"
```
Comment (extend1994) [inline ajax/libs/pickmeup/package.json]: (It's Memo not request, I will request review from Peter to make final decision)
Though the current pattern can get the files we need, I prefer a simpler one: `+(js|css)/*pickmeup*.+(js|css|map)`.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

File: ajax/libs/PickMeUp/package.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+      },
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "license": "0BSD"
```
Comment (PeterDaveHello) [inline ajax/libs/PickMeUp/package.json]: Is `0BSD` typo?

**Comment by PeterDaveHello** [comment]
@sashberd please always **test** the auto-update locally, you can simply delete all the versions and fetch them by running the auto-updater, to verify if the result looks look.

**Pull request merged**